package try_catch.Task1;

public class Main {
    public static void main(String[] args) throws Exception{
        Math math = new Math();
        math.firstMethod();
        math.secondMethod();
        math.thirdMethod();
    }
}
